install.sh

```diff
@@ -91,12 +120,7 @@ function install() {
             sudo systemctl stop "$SERVICE"
         fi
         echo "creating '/etc/systemd/system/$SERVICE$SERVICE_EXTENSION'"
-        sudo cp -f "$SERVICES_DIR/$SERVICE$SERVICE_EXTENSION" "/etc/systemd/system/$SERVICE$SERVICE_EXTENSION" > "/dev/null"
-        sudo systemctl daemon-reload
-        echo "enabling [$SERVICE]"
-        sudo systemctl enable "$SERVICE"
-        echo "starting [$SERVICE]"
-        sudo systemctl start "$SERVICE"
+        cat "$SERVICES_DIR/$SERVICE$SERVICE_EXTENSION" | sed -e "s/%DIRECTORY%/${INSTALL_DIRECTORY//\//\\/}/" | sudo tee "/etc/systemd/system/$SERVICE$SERVICE_EXTENSION" > "/dev/null"
```

Comment:
Unfortunately, this places an incorrect path in the systemd unit, and still does not install the file to the correct directory.

-   Packaged systemd files belong in `/usr/lib/systemd/system`, not `/etc/systemd/system`. You already did that correctly in other parts of the script, it's just wrong here.

-   I suggest not using `${INSTALL_DIRECTORY}` as a replacement for `/usr/local/bin` but rather `/usr/local`, it would be more consistent with other build systems as well (see my comparison with Autotools below). Then we can use the variable here to write to `${INSTALL_DIRECTORY}/etc/systemd/system/$SERVICE$SERVICE_EXTENSION` instead.

-   The resulting systemd file should still be referring to /usr/bin/fw-fanctrl but instead with `--install-dir "${pkgdir}/usr/bin"` during packaging, it will contain the directory path used to prepare the package installation files and fail to work. For me that would look like this:

    ```
    ExecStart=/usr/bin/python3 "/home/icedream/Documents/Source/Git/aur.archlinux.org/fw-fanctrl-git/pkg/fw-fanctrl-git/usr/bin/fw-fanctrl" --run --no-log
    ```

    Make/Autotools normally use two separate variables `PREFIX=/usr` and `DESTDIR=${pkgdir}/usr` (where our `DESTDIR` is `INSTALL_DIRECTORY`) for this, so that's a good idea to implement here, too.